xfoil_module/routines.py

```diff
@@ -102,12 +93,12 @@ def set_input(_in):
     return reynolds, x_transition
 
 
-def load_airfoil(airfoil_file):
+def load_airfoil(xf, airfoil_file_path):
     """
     Loads airfoil geometry data from .dat file
     """
-    logger.info(airfoil_file.local_path)
-    with open(airfoil_file.local_path) as f:
+    logger.info("Loaded airfoil geometry.")
```

Comment:
I'd move this to the line after loading has succeeded to avoid confusing errors later where it hasn't loaded but the logs say it has. You could alternatively say "Loading airfoil geometry" to indicate it's about to take place